src/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "compilerOptions": {
+    "esModuleInterop": true,
+    "jsx": "react-native",
+    "lib": ["es2015"],
+    "moduleResolution": "node",
+    "strict": true,
+    "target": "es5"
```

Comment:
we may want to consider adding `noImplicitAny: true`. not necessarily a priority if we want to get this released asap, but it will make for cleaner code in the long run.

Reply:
Added, didnt have any so wasnt a big change